plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -685,6 +688,11 @@ private ConnectorTableMetadata doGetTableMetadata(ConnectorSession session, Sche
         // Partition Projection specific properties
         properties.putAll(partitionProjectionService.getPartitionProjectionTrinoTableProperties(table));
 
+        String extraProperties = table.getParameters().get(EXTRA_PROPERTIES);
+        if (extraProperties != null) {
+            properties.put(EXTRA_PROPERTIES, mapJsonCodec(String.class, String.class).fromJson(extraProperties));
```

Comment:
What if hive/spark or a different system add some additional properties - is it possible for us to capture all of them ?

Reply:
I'm not sure what you mean here?

Reply:
Say if we Hive/Spark adds some additional properties - it won't be visible here right ? Since this JsonString - captures only the extra-properties configured via Trino - Is this intentional or can we be a bit open about the properties added by other system ?

Reply:
I see. Yes that is correct. It was intentional as we didn't have a use case for that. Do we want to add that as part of this PR?